src/lib/types.ts

```diff
@@ -715,6 +719,7 @@ export interface WorkerInstance {
 }
 
 export interface WorkerNode extends WorkerInstance, Node {
+  id?: string | undefined | null;
```

Comment:
is this property necessary for this PR?

Reply:
not really, but if it had been present it would have allowed to detect an issue in https://github.com/BuilderIO/partytown/blob/main/src/lib/web-worker/worker-node.ts#L62 since regexp/string is not directly comparable to string

Reply:
Ok, thanks